pkgs/development/tools/continuous-integration/concourse/default.nix

```diff
@@ -23,21 +23,25 @@ let
 
   resources = import ./assets/resources { inherit callPackage fly; };
 
-  # TODO: Set this
-  worker_vesion = "";
```

Comment:
We did this I assume? I don't see where though

Reply:
Worker version is now tied to the concourse version now. The unified version string is `github.com/concourse/concourse.Version`.

Reply:
OK 👍